### ISSUE TITLE
The "Sort by" modal not really readable in darkmode

### ISSUE THREAD

**Issue by Shagequi** [nextcloud/android#6195]
Hi, 

The "Sort by" dialog is not easily readable on darkmode with a white color : 

![Capture d’écran 2020-06-02 à 14 50 22](https://user-images.githubusercontent.com/7050479/83522114-7cb3a900-a4e0-11ea-9dab-8cea56765206.png)

I think the font needs to be lighter

Based on master.

**Comment by AndyScherzinger** [comment]
Isn't this also related to the chosen server color (like the drawer menu issue you opened?)

**Comment by Shagequi** [comment]
Probably yes, I'm watching

**Comment by Shagequi** [comment]
![Capture d’écran 2020-06-02 à 16 37 57](https://user-images.githubusercontent.com/7050479/83533105-6eb95480-a4ef-11ea-9208-89bbda56dfb7.png)


No it's not, even if I change the fallback color.
This issue need another PR

**Comment by Shagequi** [comment]
@AndyScherzinger I have to apologize, you were half-right 😅 

The `element_fallback_color` was related to this issue, yes, but not completely. 
It was also due to `elementColor` not used in case of  "sort-by selection"

So i created the same PR for both issues : https://github.com/nextcloud/android/pull/6198

**Comment by AndyScherzinger** [comment]
Different side of the issue, try dark theme with black color... Not even visible 😑

**Comment by Shagequi** [comment]
I'll watch this

**Comment by Shagequi** [comment]
![Capture d’écran 2020-06-03 à 10 52 39](https://user-images.githubusercontent.com/7050479/83616632-5e08ed00-a588-11ea-9dc7-8285de95cf16.png)

**Comment by JorisBodin** [comment]
The "popupTheme" is also wrong if follow this logic.
![image](https://user-images.githubusercontent.com/2796439/83644612-c1f3db80-a5b1-11ea-864d-0b86eb113382.png)
